sbt/src/sbt-test/site/fopConfig/test

```diff
@@ -1,3 +1,3 @@
 # test PDF output generation with the pdf task and a custom FOP config
 > laika:pdf
-$ exists target/docs/site-fopConfig-0.1.pdf
\ No newline at end of file
+$ exists target/docs/site-fopconfig-0.1.pdf
```

Comment:
I ran the tests locally and they failed. Are you by chance using a case-insensitive file system?